### PR TITLE
Upgrade to arikawa v2

### PR DESCRIPTION
**Pull request by miacate**
This is a quick and dirty upgrade to Arikawa v2  
I may continue to work on DM support behind the scenes  
But my go skills are minimal at best so it could be a while

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -52,7 +53,12 @@ func main() {
 
 	switch {
 	case token != "":
-		ses, err = session.New(token)
+		if os.Getenv("BOT") != "" {
```
Comment (diamondburned) [inline main.go]: Is there a reason why `TOKEN="Bot abc"` wouldn't work and would need `BOT="*"` instead?

**Review by diamondburned — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (miacate) [reply main.go]: None at all it seems, I can revert this part if you'd like.

Comment (diamondburned) [reply main.go]: That would be nice.

Comment (miacate) [reply main.go]: There you go.

**Pull request merged**